extensions/common/sql/sql-core/src/testFixtures/java/org/eclipse/edc/sql/testfixtures/PostgresqlEndToEndInstance.java

```diff
@@ -42,18 +42,16 @@ static void createDatabase(String participantName) {
 
         var extensionsFolder = TestUtils.findBuildRoot().toPath().resolve("extensions");
```

Comment:
at this point the `createDatabase` could just create the database and leave the bootstrapper to create tables and indexes